Bottom: b99badef4f876bc7e1fc9eefab867b30f0916e72
Top:    cdeacdb22aaeb9fbf83cb4844afd0049db5721db
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 19:08:37 +0100

Refresh of convert-board-objects-to-glist

---

diff --git a/src/puller.c b/src/puller.c
index 488748d..1c655e1 100644
--- a/src/puller.c
+++ b/src/puller.c
@@ -2561,7 +2561,6 @@ trace_print_lines_arcs (void)
 static int
 GlobalPuller(int argc, char **argv, int x, int y)
 {
-  int i;
   int select_flags = 0;
 
   setbuf(stdout, 0);
